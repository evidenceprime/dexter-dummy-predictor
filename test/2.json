{
  "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin bibendum. This motility assessment was repeated in a new preparation from the same semen sample. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin bibendum.",
  "sections": [
    {"name": "methods", "start": 73, "end": 159}
  ]
}
