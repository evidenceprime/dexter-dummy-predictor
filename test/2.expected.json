{
  "mentions": [
    {
      "id": 0,
      "tag": "PERSON",
      "offsets": [
        0
      ],
      "words": [
        "Lorem"
      ]
    },
    {
      "id": 2,
      "tag": "PERSON",
      "offsets": [
        160
      ],
      "words": [
        "Lorem"
      ]
    }
  ],
  "equivalences": [
    {
      "id": 0,
      "tag": "PERSON",
      "items": [
        {
          "ids": [
            0,
            2
          ],
          "text": "Lorem"
        }
      ]
    }
  ]
}